### DISCUSSION TITLE
Welcome to LynxKite Discussions!

### DISCUSSION THREAD

**Discussion by darabos** [lynxkite/lynxkite#113] [General]
## 👋  Welcome!
GitHub Discussions is the right place if you want to talk about LynxKite or graph data science in general. Whether your project focuses on academic research or commercial applications, we're always happy to learn and share. Everyone is welcome.

**Comment by ghost** [comment]
Hi!
I am trying to access another Neo4J database than the default one. I have tried adding "use xxxx match ... " and also tried changing the url to neo4j://servername:7687/db/xxxx   (xxxx being the database I wish to use) but neither option works for me.
Is there another way? Thanks!!

**Reply by darabos** [reply]
Oops, we didn't consider this case and the choice of database is not exposed on the UI. It's easy to add though! I'll send a PR.

**Reply by ghost** [reply]
Great! That would be very useful for us. Thanks!!!

**Reply by darabos** [reply]
PR #404 adds a _database_ parameter. If you can build LynxKite yourself, you can try it now. Or we can put it in a release next week.

This feature uses the Neo4j Spark Connector library, which at one point [says](https://neo4j.com/docs/spark/current/configuration/#_neo4j_driver_options) _"The driver allows to define the database in the URL [...]"_. But I can't figure out either how to do that! Some old examples use the `/db/xxxx` syntax that you've tried. Maybe you could try `neo4j://servername:7687?db=xxxx`? No idea if that works.

**Reply by ghost** [reply]
Awesome! Thanks!!
I'll wait for the new build. Having it in the docker image would be extra helpful as that is what I use today.
Once again, many thanks for acting on this so quickly! 👍😃

**Reply by ghost** [reply]
Hi Daniel,
Any chance you would be able to merge the Neo4J Database PR and build a fresh docker image?
It would help us a lot as we do not have the necessary build environment ourselves.
Many thanks!
/Mats

**Reply by darabos** [reply]
Thanks for the reminder! I've pushed out a release! You can pull `lynxkite/lynxkite:5.3.1` to be sure to get it. Sorry for the delay!

**Reply by ghost** [reply]
Thank you! That is awesome!!
/Mats
